### DISCUSSION TITLE
Can't enable feature flags on Windows Server 2012 on Erlang 26

### DISCUSSION THREAD

**Discussion by wesley0172** [rabbitmq/rabbitmq-server#11241] [Other] [answered]
### Describe the bug

When i want to enable all feature flags in the RabbitMQ Command Prompt with command: rabbitmqctl enable_feature_flag all
These errors appear: 
![image](https://github.com/rabbitmq/rabbitmq-server/assets/134276962/130aeec7-2680-43ca-af48-46ba086864da)

I've got this same problem on multiple servers. Enabling feature flags from the management console doesn't work too

### Reproduction steps

1. Have RabbitMQ Server 3.13.0 and Erlang 26.x installed
2. Open RabbitMQ Command Prompt
3. Run command: rabbitmqctl enable_feature_flag all
...

### Expected behavior

I expect the feature flags to be enabled without an error

**Comment by mkuratczyk** [comment, marked as answer]
This is not a RabbitMQ bug but rather an incompatibility between an old unsupported version of Windows and a new version of Erlang: https://github.com/erlang/otp/issues/8427

If you are running into this issue on a supported Windows version, please report it to the Erlang team.

**Comment by michaelklishin** [comment]
The RabbitMQ team has dropped testing on Windows Server 2012 a few years ago. Even Microsoft no longer supports that ≈ 13 year old distribution. I suspect neither do the Erlang/OTP maintainers.

**Comment by michaelklishin** [comment]
Ah, now I see that @mkuratczyk has linked to a direct response from the Erlang maintainers who confirm that they require a certain maintenance release of Windows 10 as the minimum.
